torch/quantization/convert_modules.py

```diff
@@ -51,6 +51,7 @@ def _observer_forward_hook(self, input, output):
     """
     self.observer(output)
 
+# TODO(jerryzh): remove_observer?
```

Comment:
Not sure if we really need it. We can instead show in the tutorial how to do a deepcopy, so that the original float module is still available.